txauthor.go

```diff
@@ -57,6 +53,22 @@ func (tx *TxAuthor) RemoveSendDestination(index int) {
 	}
 }
 
+func (tx *TxAuthor) SendDestination(atIndex int) *TransactionDestination {
+	return &tx.destinations[atIndex]
```

Comment:
There should be a check here for `atIndex` and this function should also return an error because it'll cause a panic if an invalid index is passed.